packages/app/src/elements/items-list.ts

```diff
@@ -657,6 +658,11 @@ export class ItemsList extends StateMixin(LitElement) {
                   icon: "time",
                   text: $l("You don't have any recently used items!"),
               }
+            : audit
+            ? {
+                  icon: "shield-check",
+                  text: noItemsTextForAudit(audit),
```

Comment:
I didn't see this pattern anywhere, but I needed a readable/easy/simple way to set different texts here, and didn't want to `: ?` a few more.

Reply:
This pattern is fine 👍